app/controllers/web/reminders_controller.rb

```diff
@@ -0,0 +1,76 @@
+class Web::RemindersController < Web::ApplicationController
```

Comment:
I don't think it's a cool name for this functionality. We can call it something like `passwords_controller` or `password_resets_controller'